Docker.DotNet/Models/CreateContainerParameters.Generated.cs

```diff
@@ -108,6 +114,12 @@ public CreateContainerParameters(Config Config)
         [DataMember(Name = "StopSignal", EmitDefaultValue = false)]
         public string StopSignal { get; set; }
 
+        [DataMember(Name = "StopTimeout", EmitDefaultValue = false)]
+        public long StopTimeout { get; set; }
```

Comment:
Yes but unfortunately this is tricky. Lets make an issue for this for next release. We need to know what the destination target expects. IE: nanoseconds, seconds, etc. Because they are not standardized I will need to handle conversion more specifically through some additional attribute.